applications/openshift/api-server/api_server_no_adm_ctrl_plugins_disabled/tests/no_disabled.pass.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# remediation = none
+
+yum install -y jq
+
+kube_apipath="/kubernetes-api-resources"
+
+mkdir -p "$kube_apipath/api/v1/namespaces/openshift-kube-apiserver/configmaps"
+
+config_apipath="/api/v1/namespaces/openshift-kube-apiserver/configmaps/config"
+
+# This file assumes that we have two routes outside the openshift/kube namespaces, and 
+# one route has rate limit enabled, and one route does not
```

Comment:
Is this comment valid for this change? Looks the same in each test proposed here, but we're not dealing with routes.